rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
         self.batch_size = self.loader._batch_size
         self.out = self.dimensions = self.torch_dtype = None
         self.len = b.getRemainingImages(self.loader._handle)
+        self.display = display
```

Comment:
Any reason we not adding the changes for multiple augmentations here in pytorch.py

Reply:
I have not added the multiple augmentation support in pytorch.py since pytorch.py and tf.py is not yet tested and no other changes except API name changes are done in both files